Mavsdk-Swift-Example/App/Menu/Connection/CommLinkStartView.swift

```diff
@@ -36,6 +36,35 @@ struct CommLinkView: View {
     }
 }
 
+struct CommLinkConnectView: View {
```

Comment:
I would move this struct to a separate file for consistency

Reply:
Good point! That's done :slightly_smiling_face: